### DISCUSSION TITLE
system structure suggestion

### DISCUSSION THREAD

**Discussion by chang-cancan** [minio/minio#20252] [Q&A]
Plan create new pool , least 400T, don't know who to choose structure.

use 4 servers, each has 8T * 16 drives , use JBOD group 4 drives to 1, then on system show 4 drive.
use 4 servers, each has 8T * 16 drives, on system show 16 drive.
use 8 servers, each has 8T * 8 drives, on system show 8 drive.

I perfer first one, but most server seems not support JBOD, does any has good suggestion? or recommend another structure?

thanks.

**Comment by klauspost** [comment]
Reach out on `sales@min.io` where we can help you with architecture. Include your use case, like object count and typical sizes, expected requests per minute and if you have any thoughts on hardware.
